src/core/edgeNetwork/validateNetworkResponseIsWellFormed.js

```diff
@@ -19,6 +19,8 @@ import { NO_CONTENT } from "../../constants/httpStatusCode";
 export default networkResponse => {
   const { statusCode, body, parsedBody } = networkResponse;
   if (
+    statusCode < 200 ||
+    statusCode >= 300 ||
```

Comment:
When there is a 400 or 500 status code, could there be a json body? If so we may want to throw an error that has those fields as properties. Now that Konductor's error responses are structured, Alloy's responses could be structured as well.

Reply:
For others reading this, I talked to @jonsnyder over BlueJeans about this. If Konductor was able to respond, then it will have a JSON body, but if it's JAG that's responding, it won't be a JSON body (it'll just be a string of text). While we could add a property on the error object that contains more details, we decided that including the response body in the error message like we're doing now is sufficient until we understand more about use cases where customers might want to programmatically react to certain errors coming back from the server.